core/logger/logger.go

```diff
@@ -65,6 +65,26 @@ func (l *Logger) SetLevel(lvl int) {
 	l.lvl = lvl
 }
 
+// syncPrefixes - syncs the logger prefixes
```

Comment:
It's private function, comment not needed

Reply:
Got it, but having comments for every function, helps in the readability of code right!